src/dbfread/version.py

```diff
@@ -13,5 +14,6 @@ def _make_version_info(version):
 
     return VersionInfo(major, minor, micro, releaselevel, 0)
 
-version = '2.1.0'
+
+version = '2.1.1'
```

Comment:
so now the version is stored here and in `setup.py`?

Reply:
Sadly, yes. The way the packaging worked before was fragile -- it imported dbfread inside setup.py in order to get access to the version defined in this module.  But you can't rely on being able to do that -- because by definition when you are installing dbfread you probably don't have dbfread installed. It only worked because the package itself was sitting in the top level directory and so was importable. Isolating the package under the `src/` directory makes that import impossible, and helps ensure that when you run the tests, you test the installed package, not the development repository.

I considered going through the `setuptools_scm` gymnastics to pull the version directly from git, like we do for pudl, but decided against it for the moment since that also affects the way files are selected for inclusion in the deployed package, and I didn't feel like trying to re-figure that out.